lib/jna/clink_jna.cc

```diff
@@ -54,10 +54,15 @@ extern "C" {
 
 namespace {
 tfrt::HostContext *getJnaHostContext() {
```

Comment:
Since `getJnaHostContext()` is supposed to be super light weight and will be called in multiple places, could we add the `inline` keyword here?

Same for `getJnaExecutionContext(...)`.